godef.go

```diff
@@ -218,8 +218,10 @@ func (o orderedObjects) Swap(i, j int)      { o[i], o[j] = o[j], o[i] }
 
 func done(obj *ast.Object, typ types.Type) {
 	defer os.Exit(0)
-	pos := types.FileSet.Position(types.DeclPos(obj))
-	fmt.Printf("%v\n", pos)
+	for o := obj; o != nil; o = o.Next {
+		pos := types.FileSet.Position(types.DeclPos(o))
+		fmt.Printf("%v\n", pos)
```

Comment:
I wonder what existing editor plugins will make of this... Perhaps we need a flag to ask for all definitions.